ecommerce/frontend/src/components/SearchInput.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './SearchInput.less'
+import { Select } from 'antd'
+import Search from 'antd/lib/input/Search'
+
+// TODO - Fetch from a common file or backend
+const categories = [
+    { key: 'electronics', title: 'Electronics' },
+    { key: 'home_and_garden', title: 'Home & Garden' },
+    { key: 'personal_care', title: 'Personal Care' },
+    { key: 'furniture', title: 'Furniture' },
+    { key: 'pet_supplies', title: 'Pet Supplies' },
+    { key: 'mens_fashion', title: "Men's Fashion" },
+    { key: 'womens_fashion', title: "Women's Fashion" },
+]
+
+const { Option } = Select
+
+function handleCategoryChange(value) {
+    console.log(`selected ${value}`)
+}
+
+const categoryDropdown = (
+    <Select defaultValue="Electronics" style={{ width: 120 }} onChange={handleCategoryChange}>
+        {categories.map(category => (
+            <Option value={category.title}>
+                {category.title} key = {category.key}
+            </Option>
+        ))}
+    </Select>
+)
```

Comment:
I would have categoryDropdown variable inside the SearchInput component, before the return.

Reply:
Also have the `handleCategoryChange` a function inside the SearchInput component that updates a 

```js
const [category, setCategory] = useState(someDefaultCategory)
```

and have the Select `defaultValue={category}` since defaultValue is only used on the first render of `Select`